packages/elements/src/components/metrics-table/metrics-table.component.ts

```diff
@@ -127,7 +132,7 @@ export class MutationTestReportTestMetricsTable<TFile, TMetric> extends LitEleme
         </td>
         <td style="width: 50px;" class="fw-bold text-center text-${coloringClass}">${valueIsPresent ? mutationScoreRounded : undefined}</td>`;
     }
-    return column.isHeader ? html`<th class="text-center">${value}</th>` : html`<td class="text-center">${value}</td>`;
+    return html`<td class="text-center ${column.isHeader ? 'fw-bold' : ''}" aria-describedby="${`tooltip-${column.key.toString()}`}">${value}</td>`;
```

Comment:
Why is this change needed? Is a `th` not desired here for some reason?

Reply:
Because it's not a table header 😁. It's very much a table data cell, just bold (this is the total number of mutants, for example, in a row). So it looks the same (bold if `isHeader`), but makes more sense semantically now

Reply:
Alright. So why do we keep the model name as `isHeader` then? 🤔